opentracing_instrumentation/client_hooks/__init__.py

```diff
@@ -53,7 +53,8 @@ def install_all_patches():
     tornado_http.install_patches()
     urllib.install_patches()
     urllib2.install_patches()
-    requests.install_patches()
+    RequestsPatcher.install_patches()
+    RequestsPatcher.set_response_handler_hook(requests_response_handler_hook)
```

Comment:
so rather than passing `requests_response_handler_hook` to `install_all_patches()`, couldn't the user just make this call above first? As I understand we're still using a global singleton here.

Reply:
Yes, the user could. So I removed `requests_response_handler_hook` from `install_all_patches` method for now.